pages/en/lb2/Deploy-for-GDPR-readiness.md

```diff
@@ -0,0 +1,212 @@
+The GDPR legislation requires that personal data is strictly controlled and that the 
+integrity of the data is maintained. This requires the data to be secured against loss 
+through system failure and also through unauthorized access or via theft of computer equipment or storage media.
+
+**LoopBack Overview**
+
+LoopBack is an open source Node.js framework that allows you to create REST APIs and connect to the backend resources, such as database and Web services.
+- Users can download LoopBack anonymously and use LoopBack without creating any accounts.
+- LoopBack itself is not runnable. The close analogy is Spring Framework or IBM Node.js SDK.
+- Customers create applications using LoopBack tooling and runtime.
+- Since LoopBack allows users to connect to backend resources that might require credentials to access, users can create a LoopBack application in such way that the credentials are stored in the configuration files as part of their LoopBack application.
+
+It is important to note that it is developer's responsibility to design and configure their LoopBack application using the [GDPR Privacy By Design](https://www.eugdpr.org/more-resources-1.html) as guidelines.
+
+##### Configuration to support Data Privacy
+It is developer's responsibility to design and configure their LoopBack application using the [GDPR Privacy By Design](https://www.eugdpr.org/more-resources-1.html) as guidelines to support Data Privacy.
+
+##### Configuration to support Data Security
+It is developer's responsibility to design and configure their LoopBack application using the [GDPR Privacy By Design](https://www.eugdpr.org/more-resources-1.html) as guidelines to support Data Security.
+
+___________________________________________________________________________________________________
+### Data Life Cycle
+
+
+#### What types of data flow through LoopBack?
+
+LoopBack is a development framework that does not collect any data and send it back to IBM.  
+The LoopBack framework facilitates the connection to backend resources, such as databases and Web services, where identity data is stored.  However, it does not store actual data.  
+
+There are two types of data that are involved in LoopBack: application configuration data and application data. 
+
+1. Application configuration data
+
+    The following files are the LoopBack application configuration data that might captures identity data,
+such as database connection information (IP addresses and credentials), in order to connect to the backend resources:
+    - `server/config.json`
+    - `server/datasources*.json`
+    - `server/datasources*.js`
+    - `server/model-config.json`
+    - `server/middleware*.json`
+
+2. Application data
+
+    LoopBack provides built-in User model that makes the LoopBack applications require authentication. However, it is not compulsary to use.  Besides, developers can create other models that involve personal data, which the LoopBack framework has no control over.  
```

Comment:
I think it'd be better to describe here how predefined models like `User` is only intended to be a template of some sort and it is the developers' responsibility to properly handle the the data